app/junitdiff

```diff
@@ -38,4 +38,5 @@ $app->setVersion('0.2.1');
 $app->setName('JUnitDiff (by Andreas Heigl and contributors)');
 
 $app->add(new \Org_Heigl\JUnitDiff\Command\DiffCommand());
```

Comment:
What's the difference between these two commands? Shouldn't there only be one command?